flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": [ "x86_64", "i386", "arm" ],
+    "skip-appstream-check": true
```

Comment:
Why did you disable appstream validation?

Reply:
By mistake, my two closed PR's have bad appstream files, no icon etc.